Rest/Controller/ContentController.php

```diff
@@ -149,16 +132,44 @@ public function getContent($contentIdList, $responseType, Request $request)
         $query = new Query();
         $query->query = new Criterion\LogicalAnd($criteria);
 
-        $contentItems = $this->searchService->findContent($query)->searchHits;
+        $contentItems = $this->searchService->findContent(
+            $query,
+            (!empty($lang) ? array('languages' => array($lang)) : array())
+        )->searchHits;
 
         $content = $this->prepareContent(array($contentItems), $request);
 
-        return new ContentDataValue($content, [
-            'responseType' => 'http',
-            'documentRoot' => $request->server->get('DOCUMENT_ROOT'),
-            'host' => $request->getSchemeAndHttpHost(),
-            'customerId' => $this->customerId,
-        ]);
+        return new ContentDataValue($content);
+    }
+
+    /**
+     * Preparing array of integers based on comma separated integers in string or single integer in string.
+     *
+     * @param string $string list of integers separated by comma character
+     *
+     * @return array
+     *
+     * @throws InvalidArgumentException If incorrect $list value is given
+     */
+    protected function getIdListFromString($string)
+    {
+        if (filter_var($string, FILTER_VALIDATE_INT)) {
```

Comment:
Nitpick: `filter_var(...) !== false` would be safer because `filter_var` 

> Returns the filtered data, or FALSE if the filter fails.

But this is OK until 0 is wrong ID :wink: